kitchen.ec2.yml

```diff
@@ -39,10 +41,4 @@ lifecycle:
         echo "NOTICE - Updating the ec2-user to keep sudo working"
         sudo chage -d $(( $( date +%s ) / 86400 )) ec2-user
         echo "NOTICE - updating ec2-user sudo config"
-        sudo chmod 600 /etc/sudoers && sudo sed -i'' "/ec2-user/d" /etc/sudoers && sudo chmod 400 /etc/sudoers
-
-        
-transport:
-  name: ssh
-  #https://github.com/neillturner/kitchen-ansible/issues/295
-  max_ssh_sessions: 2
```

Comment:
Not sure why, but I think combining the `transport` blocks is failing actions:
![Screen Shot 2023-04-06 at 12 45 41 PM](https://user-images.githubusercontent.com/11844975/230443420-1c7c487f-5bfa-40da-b3b1-c98263441336.png)


While having separate `transport` blocks seems to work:
![Screen Shot 2023-04-06 at 12 45 24 PM](https://user-images.githubusercontent.com/11844975/230443360-3825260f-01ca-44fe-8a73-22e19d84a41d.png)

Reply:
Logged [an issue](https://github.com/mitre/redhat-enterprise-linux-7-stig-baseline/issues/186)